itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTables.java

```diff
@@ -308,6 +316,9 @@ public void testFailoverDuringDump() throws Throwable {
             .run("select rank from t2 order by rank")
             .verifyResults(new String[]{"10", "11"});
 
+    db = replica.getDatabase(replicatedDbName);
+    assertFalse(MetaStoreUtils.isTargetOfReplication(db));
```

Comment:
Wouldn't this mean that there will not be any rollback?

Reply:
In case of rollback, repl.targer.for would be again set in incremental load.

Reply:
It might take a while before rollback can kick in, meanwhile if background threads run, wouldn't they mess up the state.

Reply:
repl.failover.endpoint db prop would restrict backgrounds threads to run for such db if rollback is initiated.